lib/Doctrine/Persistence/Mapping/AbstractClassMetadataFactory.php

```diff
@@ -91,7 +91,7 @@ public function setCacheDriver(?Cache $cacheDriver = null)
             throw new BadMethodCallException('Cannot convert cache to PSR-6 cache');
```

Comment:
should this exception be removed entirely ? AFAICT, `CacheAdapter` accepts a `Cache` as its argument (unless the code in doctrine/cache is wrong)

Reply:
You are correct, this was left over from the previous compatibility layer. I've created #182 to fix this.